velox/expression/tests/ExprTest.cpp

```diff
@@ -1273,6 +1273,27 @@ TEST_F(ExprTest, switchExpr) {
   assertEqualVectors(expected, result);
 }
 
+TEST_F(ExprTest, switchExprWithNull) {
+  vector_size_t size = 1'000;
+  auto vector = makeRowVector(
+      {makeFlatVector<int32_t>(
+           size, [](auto row) { return 7; }, [](auto row) { return row % 2; }),
```

Comment:
Any particular reason not to use nullEvery(2) here?

Reply:
No particular reason. Just to inject null at odd and ease verification below. I will add comments.